cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -397,8 +474,10 @@ def test_cluster_section_from_file(mocker, config_parser_dict, expected_dict_par
         ("fsx_settings", "test1", None, "Section .* not found in the config file"),
     ],
 )
-def test_cluster_param_from_file(mocker, param_key, param_value, expected_value, expected_message):
-    utils.assert_param_from_file(mocker, CLUSTER, param_key, param_value, expected_value, expected_message)
+def test_sit_cluster_param_from_file(
```

Comment:
We should add a similar test for hit_cluster, to be sure only the supported parameters are available.

We can move the common ones in another test and have two tests for the two specific models with the subset of specific parameters.

Reply:
Then we need to add test_section_queue.py and test_section_compute_resource.py 

They are very important to be sure to have defined the right parameters in the mapping, with the right validators.

Reply:
`` test_sit_cluster_param_from_file`` is already implemented

Reply:
`test_section_queue` and `test_section_compute_resource` are also already present.

Reply:
Sorry I missed them because they are not part of this patch.